fenix/app/metrics.yaml

```diff
@@ -143,13 +143,14 @@ events:
           bookmarks, desktop_view_off, desktop_view_on, downloads,
           find_in_page, forward, history, new_tab, open_in_app, open_in_fenix,
           quit, reader_mode_appearance, reload, remove_from_top_sites,
-          save_to_collection, set_default_browser, settings, share, stop, and
-          sync_account.
+          save_to_collection, set_default_browser, settings, share, stop,
+          sync_account, and print_content.
```

Comment:
I'm not sure if we need data review for updating existing items?

Reply:
I wasn't entirely sure either - I had another related bug that did need a data review, so I tagged it in that review, just in case.

Reply:
Updated with data review link!